pvlib/pvarray.py

```diff
@@ -225,24 +225,54 @@ def adr_wrapper(xdata, *params):
         return popt
 
 
-def _infer_k_huld(cell_type, pdc0):
+def _infer_k_huld(cell_type, pdc0, k_version):
+    r"""
+    Get the EU JRC updated coefficients for the Huld model.
+
+    Parameters
+    ----------
+    cell_type : str
+        Must be one of 'csi', 'cis', or 'cdte'
+    pdc0 : numeric
+        Power of the modules at reference conditions [W]
+    k_version : str
+        Either '2011' or '2025'.
+
+    Returns
+    -------
+    tuple
+        The six coefficients (k1-k6) for the Huld model, scaled by pdc0
+    """
     # from PVGIS documentation, "PVGIS data sources & calculation methods",
     # Section 5.2.3, accessed 12/22/2023
     # The parameters in PVGIS' documentation are for a version of Huld's
     # equation that has factored Pdc0 out of the polynomial:
     #  P = G/1000 * Pdc0 * (1 + k1 log(Geff) + ...) so these parameters are
     # multiplied by pdc0
-    huld_params = {'csi': (-0.017237, -0.040465, -0.004702, 0.000149,
-                           0.000170, 0.000005),
-                   'cis': (-0.005554, -0.038724, -0.003723, -0.000905,
-                           -0.001256, 0.000001),
-                   'cdte': (-0.046689, -0.072844, -0.002262, 0.000276,
-                            0.000159, -0.000006)}
+    if k_version == '2011':
+        huld_params = {'csi': (-0.017237, -0.040465, -0.004702, 0.000149,
+                               0.000170, 0.000005),
+                       'cis': (-0.005554, -0.038724, -0.003723, -0.000905,
+                               -0.001256, 0.000001),
+                       'cdte': (-0.046689, -0.072844, -0.002262, 0.000276,
+                                0.000159, -0.000006)}
+    elif k_version == '2025':
+        # Updated coefficients from EU JRC paper
+        huld_params = {'csi': (-0.017162, -0.040289, -0.004681, 0.000148,
+                               0.000169, 0.000005),
+                       'cis': (-0.005521, -0.038576, -0.003711, -0.000901,
+                               -0.001251, 0.000001),
+                       'cdte': (-0.046477, -0.072509, -0.002252, 0.000275,
+                                0.000158, -0.000006)}
+    else:
+        raise ValueError(f'Invalid k_version={k_version}: must be either '
+                         '"2011" or "2025" as a string')
     k = tuple([x*pdc0 for x in huld_params[cell_type.lower()]])
     return k
 
 
-def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None):
+def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None,
+         k_version=None):
```

Comment:
```suggestion
         k_version='2011'):
```
Why not make it explicit here?